### DISCUSSION TITLE
GIMP

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#9] [Requests] [answered]
GIMP website: https://www.gimp.org/

GIMP privacy policy: https://gimp.cc/privacy-policy/

Logo: https://upload.wikimedia.org/wikipedia/commons/4/45/The_GIMP_icon_-_gnome.svg

Description: "GIMP (/ɡɪmp/ GHIMP; GNU Image Manipulation Program) is a free and open-source raster graphics editor used for image manipulation (retouching) and image editing, free-form drawing, transcoding between different image file formats, and more specialized tasks.

GIMP is released under GPL-3.0-or-later license and is available for Linux, macOS, and Microsoft Windows."

**Comment by mayrinck** [comment, marked as answer]
Oh yeah, GIMP was on my personal list over here. I will add it as soon as possible. Thank you for submitting!

**Comment by mayrinck** [comment]
Gimp is now available: https://www.microsoft.com/store/apps/9NJP9724LT35

**Reply by mayrinck** [reply]
GIMP may need to be re-uploaded, just found a critical bug in the initialization... For now, it will be buggy there, beware!

**Reply by JuliaAgostino** [reply]
> GIMP may need to be re-uploaded, just found a critical bug in the initialization... For now, it will be buggy there, beware!

Okay, good to know.

**Comment by JuliaAgostino** [comment]
Just out of curiosity, what happened to GIMP? Was the latest version not packageable or something like that?

**Reply by mayrinck** [reply]
Well, what happened is that when GIMP does startup, it must read a certain range of .exe files that are basically the filters and settings of the app. The package version, for some reason, can't find those .exe files when GIMP loads, so it simply doesn't work properly...

I hope this is just a permission issue, because this can be added to the package during conversion... If the problem is different, maybe there will be no solution...

**Reply by JuliaAgostino** [reply]
> Well, what happened is that when GIMP does startup, it must read a certain range of .exe files that are basically the filters and settings of the app. The package version, for some reason, can't find those .exe files when GIMP loads, so it simply doesn't work properly...
> 
> I hope this is just a permission issue, because this can be added to the package during conversion... If the problem is different, maybe there will be no solution...

Well, I hope that GIMP can make a return to the Microsoft Store at some point.
